qrenderdoc/3rdparty/toolwindowmanager/ToolWindowManager.cpp

```diff
@@ -490,7 +490,7 @@ void ToolWindowManager::restoreState(const QVariantMap &dataMap) {
     wrapper->restoreState(windowData.toMap());
     wrapper->updateTitle();
     wrapper->show();
-    if(wrapper->windowState() && Qt::WindowMaximized)
+    if(wrapper->windowState() & Qt::WindowMaximized)
```

Comment:
This change will need to go upstream to the [separate toolwindowmanager fork](https://github.com/baldurk/toolwindowmanager), so can you revert it in this CL?

I'll sort out making the change in that repo and merging it in here so you don't have to make a second PR. But just if we do it that way then all the changes to `qrenderdoc/3rdparty/toolwindowmanager` will just be updating to a particular hash in the other repo, instead of also containing mixed changes that originate here.